webpay/pay/views.py

```diff
@@ -157,7 +158,12 @@ def configure_transaction(request, data=None):
     else:
         sim = (request.session['notes']['pay_request']['request']['simulate']
                if is_simulation else None)
-        return {'status': 'ok', 'simulation': sim}
+        client_trans_id = 'client-trans:{u}'.format(u=uuid.uuid4())
```

Comment:
the session trans_id will look like ``webpay:abc123`` and the other will look like ``client-trans:abc123`` so it should be ok